data-raw/scrape.R

```diff
@@ -83,7 +83,7 @@ rgb <- df %>%
 
 team_colors <- hex %>%
   bind_rows(select(rgb, -color_rgb)) %>%
-  mutate(color_hex = tolower(color_hex))
+  mutate(color_hex = tolower(color_hex)) %>%
```

Comment:
Nice catch!

Reply:
Can you also please add yourself as a `ctb` in the `DESCRIPTION` file? And update the `NEWS.md`?